__tests__/batching.test.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import { render, cleanup, act } from '@testing-library/react/pure';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Does the linter cry because of the linking/mocking here?

Reply:
It cries because react-easy-state is not installed as a node package. It is only linked, and eslint cannot see that sadly.